satpy/tests/reader_tests/test_seviri_l1b_icare.py

```diff
@@ -137,24 +148,60 @@ def test_load_dataset_ir(self):
         for v in datasets.values():
             self.assertEqual(v.attrs['calibration'], 'brightness_temperature')
 
-    def test_area_def(self):
+    def test_area_def_lores(self):
         """Test loading all datasets from an area of interest file."""
         r = load_reader(self.reader_configs)
         loadables = r.select_files_from_pathnames([
-            'GEO_L1B-MSG1_2004-12-29T12-15-00_G_VIS08_V1-04.hdf',
+            'GEO_L1B-MSG1_2004-12-29T12-15-00_G_VIS08_V1-04.hdf'
         ])
         r.create_filehandlers(loadables)
         datasets = r.load(['VIS008'])
-        test_area = {'area_id': 'geosmsg',
-                     'width': 10,
-                     'height': 300,
-                     'area_extent': (-5567248.2834071,
-                                     -5570248.6866857,
-                                     -5537244.2506213,
-                                     -4670127.7031114)}
         for v in datasets.values():
```

Comment:
There should be just one dataset loaded right? Why loop?

Reply:
Because I had no idea what I was doing ;-) It was code copied from elsewhere that I didn't understand. Now fixed!